conf/solr/start.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+# Default to dev env
+ENV=${ENV:-dev}
+
+ln -sf /etc/solr/conf/solrconfig-$ENV.xml /etc/solr/conf/solrconfig.xml
+
+# On dev, don't use haproxy; make tomcat serve directly to 8983
+if [ "$ENV" = "dev" ] ; then
+    sed -i 's/8080/8983/g' /var/lib/tomcat7/conf/server.xml;
```

Comment:
this could bite us eventually? no way of knowing (while within server.xml) not to add values 8080 or 8983 :P